packages/ord-connect/src/components/SelectWalletModal/index.tsx

```diff
@@ -72,52 +75,49 @@ export function SelectWalletModal({
     [disconnectWallet],
   );
 
-  const onConnectUnisatWallet = useCallback(
-    async ({ readOnly }: { readOnly?: boolean } = {}) => {
-      try {
-        // Reset error message
-        setErrorMessage("");
-        const unisat = await getUnisatAddresses(network, readOnly);
+  const onConnectUnisatWallet = useCallback(async () => {
```

Comment:
Was this changed intentionally? This will cause a regression

Reply:
ohh, I imported the wrong wallet function, it should be left as is, will update it

Reply:
updated